entity-web/src/main/java/eu/europeana/entity/web/controller/UsageStatsController.java

```diff
@@ -0,0 +1,57 @@
+package eu.europeana.entity.web.controller;
+
+import eu.europeana.api.common.config.swagger.SwaggerSelect;
+import eu.europeana.api.commons.definitions.statistics.entity.EntityMetric;
+import eu.europeana.api.commons.definitions.vocabulary.CommonApiConstants;
+import eu.europeana.api.commons.error.EuropeanaApiException;
+import eu.europeana.api.commons.web.exception.HttpException;
+import eu.europeana.entity.stats.exception.UsageStatsException;
+import eu.europeana.entity.stats.vocabulary.UsageStatsFields;
+import eu.europeana.entity.web.controller.exception.EntityApiRuntimeException;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.Date;
+
+@Controller
+@Api(tags = "Discovery API")
```

Comment:
we should have an own grouping for stats

Reply:
done - changed to usage Statistics api